assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -253,288 +253,277 @@ db.schema.flyway.scripts.prefix=
 # Suffix of migration scripts.
 db.schema.flyway.scripts.suffix=.sql
 
-# Separator of version from the other part of script name.
+# Separator of `version` from the other parts of script name.
 db.schema.flyway.scripts.version_separator=__
 
-# Locations where to search migration scripts.
+# Locations where to search for migration scripts.
 db.schema.flyway.scripts.locations=classpath:che-schema
 
-### Kubernetes Infra parameters
+### Kubernetes infrastructure parameters
 
-# Configuration of Kubernetes client master URL that Infra will use.
+# Configuration of Kubernetes Client master URL that infrastructure will use.
 che.infra.kubernetes.master_url=
 
-# Boolean to configure Kubernetes client to use trusted certificates.
+# Boolean to configure Kubernetes Client to use trusted certificates.
 che.infra.kubernetes.trust_certs=false
 
-# Kubernetes cluster domain. If not set, svc names will not contain information about the cluster domain.
+# Kubernetes cluster domain. If not set, svc names won't contain information about the cluster domain.
```

Comment:
to avoid avoid future tense, can you say, `If not set, svc names do not/cannot contain information about the cluster domain.`